### PR TITLE
add support for weather api auth token

### PR DESCRIPTION
**Pull request by nsherred**
Openweathermap now requires an api token to access their api.

### CODE REVIEW

File: src/client.js [hunk 1]
```diff
@@ -54,7 +56,7 @@ var client = function (context) {
             }
 
             return Promise.props({
-                current:  methods.current(params),
+                current: methods.current(params),
```
Comment (plouc) [inline src/client.js]: Can you keep it aligned please ? Sorry, I should have added eslint rules for all mozaïk modules…

File: src/config.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+var convict = require('convict');
+var check   = require('validator').check;
+
+var configurator = convict({
+    weather: {
+        token: {
+            doc: 'The weather API token',
```
Comment (plouc) [inline src/config.js]: Same here

File: src/config.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+            default: null,
+            format: String,
+            env: "WEATHER_SERVICE_KEYPATH"
```
Comment (plouc) [inline src/config.js]: I also prefer to keep consistent quotes, using single quote

**Comment by nsherred** [comment]
I've updated the code so that its format is inline with the rest of the project.

**Comment by plouc** [comment]
@Nsherred thanks a lot for your contribution, I'll try to release the package as soon as possible

**Pull request merged**